### DISCUSSION TITLE
Data Consistency (AU) : feature does not correspond to edge in topological structure

### DISCUSSION THREAD

**Discussion by Dnoack49** [INSPIRE-MIF/helpdesk-validator#1243] [Ideas]
Dear all, 

When validating the following [AdministrativeBoundary-Dataset](https://inspire.brandenburg.de/services/au_bdlm_wfs?service=WFS&version=2.0.0&request=GetFeature&Count=10&typenames=au:AdministrativeBoundary)

I receive the following Error-Message in the Module "_Conformance class: Dataconsistency, Administrative Units / Boundary/ au-dc.f.1 The geometry of each instance administrattive boudnary corresponds to an edge in the topological structure formed by the complete boundary graph, including the boundaries of all levels_"

`
The feature does not correspond to an edge in the topological structure formed by the complete boundary graph of all AdministrativeUnit features. It is not on the boundary of any AdministrativeUnit in the dataset.`

![grafik](https://github.com/INSPIRE-MIF/helpdesk-validator/assets/8046960/3820660f-2779-4fa4-9b42-19049a6ac04d)

When looking at the example object  with ID _AdministrativeBoundary_DEBBATP00000lDIz_ :

The AdministrativeBoundary however is on the boundary of the AdministrativeUnit

![grafik](https://github.com/INSPIRE-MIF/helpdesk-validator/assets/8046960/a2c7b15f-a8b8-4150-91bc-77d99b5a01cd)

Or is the reason the nilReason-Value in the adminUnit-Reference?

I would be thankful for an advice.

Kind regards

**Comment by fabiovinci** [comment]
Dear @Dnoack49,

thank you for the issue, we will have a look asap.

**Comment by Dnoack49** [comment]
Any news on this issue?

**Comment by fabiovinci** [comment]
Dear @Dnoack49,

sorry for the late reply.
The issue is due to the fact that you are validating only the "AdministrativeBoundary" feature type and the validator looks for an "AdministrativeUnit" inside the dataset itself for the check about the boundary requirements.

Regarding the `admUnit` attribute, you should populate it to document the association of the boundary with the unit, but since it is voidable the validator doesn't check it.

We are considering introducing a manual check in the [related test](https://github.com/inspire-eu-validation/data-au/blob/master/au-dc/boundary.md), to handle situations like yours.

**Comment by fabiovinci** [comment]
Dear all,
since this issue requires the development of new features that are currently unplanned, it will be moved to the related discussion section.
